### PR TITLE
Perform more docs smithing

### PR DESCRIPTION
**Pull request by mavam**
This PR contains a potpourri of docs-related improvements.

Most notably, we added [giscus](https://giscus.app/) commenting to the blog, using [this tutorial](https://dev.to/m19v/how-to-add-giscus-comments-to-docusaurus-439h). I enabled comments for the last two Zeek blogs.

### CODE REVIEW

File: web/src/theme/BlogPostItem/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { useBlogPost } from '@docusaurus/theme-common/internal'
+import BlogPostItem from '@theme-original/BlogPostItem';
+import GiscusComponent from '@site/src/components/GiscusComponent';
+import useIsBrowser from '@docusaurus/useIsBrowser';
+
+export default function BlogPostItemWrapper(props) {
+  const { metadata, isBlogPostPage } = useBlogPost()
+  const isBrowser = useIsBrowser();
+
+  const { frontMatter, slug, title } = metadata
```
Comment (dit7ya) [inline web/src/theme/BlogPostItem/index.js:11]:
```suggestion
  const { frontMatter } = metadata
```

**Review by dit7ya — COMMENTED** [review]
I have not used Giscus before but this looks simple enough to me, expect the comments. I guess we will have to just tune Giscus a little bit as we continue.

File: web/src/components/GiscusComponent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export default function GiscusComponent() {
+  const { colorMode } = useColorMode();
+
+  return (
+    <Giscus
+      repo="tenzir/tenzir"
+      repoId="MDEwOlJlcG9zaXRvcnk5MzIzMTQ"
+      category="Blog"
+      categoryId="DIC_kwDOAA452s4CX0IU"
+      mapping="url"
```
[the remark this answers is not in the document]
Comment (dit7ya) [reply web/src/components/GiscusComponent/index.tsx:14]: I just looked at the Giscus API here https://giscus.app/ at the section `Page ↔️ Discussions Mapping`.

I think the blogpost title looks/reads better than the URL for the discussion title.

[the remark this answers is not in the document]
Comment (mavam) [reply web/src/components/GiscusComponent/index.tsx]: I switched to `title`.

**Pull request merged**